.github/workflows/payload.yaml

```diff
@@ -2,7 +2,7 @@ name: Publish Enclave CC CI payloads for Confidential Containers
 on:
   push:
     branch:
-      - main
+      - 'main'
```

Comment:
I think both 'main' and main should work, but I think `branch` should actually be `branches`, as shown here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters

Reply:
thanks, I missed that! fixed.